src/boot/efi/boot.c

```diff
@@ -812,12 +828,26 @@ static BOOLEAN menu_run(
                 case KEYPRESS(0, 0, 'p'):
                 case KEYPRESS(0, 0, 'P'):
                         print_status(config, loaded_image_path);
-                        refresh = TRUE;
+                        clear = TRUE;
                         break;
 
                 case KEYPRESS(EFI_CONTROL_PRESSED, 0, 'l'):
                 case KEYPRESS(EFI_CONTROL_PRESSED, 0, CHAR_CTRL('l')):
-                        refresh = TRUE;
+                        clear = TRUE;
+                        break;
+
+                case KEYPRESS(0, 0, 'r'):
+                        if ((INT64)config->console_mode_efivar < ST->ConOut->Mode->MaxMode &&
+                            ST->ConOut->Mode->Mode == ST->ConOut->Mode->MaxMode - 1) {
+                                config->console_mode_efivar = CONSOLE_MODE_KEEP;
+                                status = StrDuplicate(L"Console mode defined by configuration file.");
```

Comment:
hmm, this is misleading, no? i mean, there might not be a config file. Hence, maybe say  "Console mode reverted to firmware default or configuration file default".